### PR TITLE
Collapse feedback

### PR DESCRIPTION
**Pull request by inexorableAce**
added some style changes to collapse headers to allow collapsed regions to be seen a bit better and be more clear that they are collapsed.

open sections
image

collapsed sections
image

### CODE REVIEW

File: src/app/dim-ui/CollapsibleTitle.scss [hunk 1]
```diff
@@ -2,20 +2,23 @@
...
   letter-spacing: 2px;
-  font-size: 14px;
+  font-size: 16px;
   display: flex;
   flex-direction: row;
   justify-content: space-between;
   align-items: center;
+  transition: 0.3s;
+  cursor: pointer;
+  border-bottom: 1px solid #ffffff;
```
[the remark this answers is not in the document]
Comment (inexorableAce) [reply src/app/dim-ui/CollapsibleTitle.scss]: these were already there... not sure we need a var for white.

File: src/app/dim-ui/CollapsibleTitle.scss [hunk 2]
```diff
@@ -2,20 +2,23 @@
 
 .title {
   flex: 1;
-  padding: 0 16px;
-  background-color: #353535;
-  color: white;
+  padding: 2px 16px;
+  background-color: transparent;
+  color: #ffffff;
```
[the remark this answers is not in the document]
Comment (inexorableAce) [reply src/app/dim-ui/CollapsibleTitle.scss]: these were already there... not sure we need a var for white.

**Comment by inexorableAce** [comment]
found some issues with alignment and scaling for vendors with the '(hidden)' text 
image

image

I suppose the scaling issue would happen anyway and it's not super detrimental. Just looks off. i'll play around see what I can do for that, the alignment thing is an easy fix.

**Comment by 48klocs** [comment]
I get the same flavor of `yarn.lock` changes - yarn `1.10.1` adds `integrity`, which prior versions didn't.

**Comment by inexorableAce** [comment]
Yeah I just noticed this... should I just exclude the yarn lock from commits?

**Comment by bhollis** [comment]
I finally had a chance to check this out. Some initial feedback:

1. It's all just a bit too much. The spreading text on hover, the hidden text, the line... it just seems too un-subtle compared to the general DIM aesthetic.
2. Having the titles have no background by default makes them look sorta broken. The white line isn't enough to give them definition, and the way they cut through the vault makes it just seem like there's a styling bug that set the wrong background.

**Comment by inexorableAce** [comment]
Fair enough. I'd argue that subtlety in actions isn't a good thing though. You want the outcome of every action a user can take to be clear before, while, and after they take it. 

Currently, these headers get clicked without a user knowing they've clicked them, and then stay closed long enough for people to ask were their stuff went. That means users are unaware that the headers can be clicked (before), click them without knowing (while), and don't know what changed or why (after). they fail all those tests. 

As for the backgrounds, I think you're right I had other changes that rectified that but it was outside the scope of this particular PR, so I removed them. The color blocking of the site in general is something I'd also like to tackle. It may seem like a good way to draw distinction, but it's actually pretty distracting to the eye and ultimately unnecessary. White space is our friend, it will give distinction without distraction. 

Long story short these changes likely are more in line with an overall visual overhaul that does things like improve contrast, reduce clutter, and remove ambiguity. Without those, you're right it seems out of place. But that's a big PR and such didn't want to make all that change in one go. 

For the time being we can close this and discuss some of the larger changes in #design till we've landed in a spot we're all comfortable with.

**Comment by bhollis** [comment]
Closing in favor of #3293
